### DISCUSSION TITLE
关于硬盘读取延迟问题的交流。

### DISCUSSION THREAD

**Discussion by lrlbh** [Chia-Network/chia-blockchain#6956] [Support]
took: 6.923975944519043. This should be below 5 seconds to minimizerisk of losing rewards.
took: 9.455993890762329. This should be below 5 seconds to minimizerisk of losing rewards.
took: 6.908346652984619. This should be below 5 seconds to minimize risk of losing rewards.
took: 7.142791986465454. This should be below 5 seconds to minimiz

我使用的USB连接硬盘。我也有这个问题，为此我换过很多台电脑，包括运行p图的5800和i7-8750h电脑，没有任务的3020e，树莓派4B，i5-6xxxU。同时也使用过linux系统和windows系统。这个问题一直都在出现，我还没有找到具体的原因。

1、目前可以确定的是原装的外置硬盘和移动硬盘不会出现该问题。
2、出现该问题的都是使用了易驱线和硬盘盒子的硬盘。
3、我使用的硬盘基本都是4T的西数紫盘和4T希捷酷鹰，我不确定是硬盘有问题还是硬盘盒和易驱线有问题。
4、个别盒子和易驱线会频繁的发生问题。

**Comment by loppefaaret** [comment]
what kind of filesystems are you using ? and are you sure that the partition holding data is sector alligned ?

**Comment by lrlbh** [comment]
![image](https://user-images.githubusercontent.com/47717790/123236994-dafa6200-d50f-11eb-9353-051e6322dbb9.png)
![image](https://user-images.githubusercontent.com/47717790/123237067-e9e11480-d50f-11eb-80a3-e6b20b2e1bc1.png)

This is machine translation and I'm not sure you can understand it.
Thank you very much for your answer, which points out a new idea for me.
I am using the default partition of Win10, do not know whether there is a problem?
At the same time, I detected a debug file that was not full. This is the number of delays per hard disk. Next I'm going to check the specified file to see if it was caused by the specified file.

这是机器翻译我不确定您可以看懂。
非常感谢你的回答，这为我指出了一个新的思路。
我是用的是win10的默认分区方式，不知道是否有问题？
同时我检测了一个没有写满的debug文件，这是每一个硬盘出现延迟的次数。下一步我打算检查指定的文件，看看是不是有指定的文件引起的。

**Reply by loppefaaret** [reply]
if the partitions was created under windows 10 - they should be aligned, and good :+1:
seeing as these are external USB drives - how are they connected to the machine ? are they connected through UBS 2.0, or USB 3.0 ? how many drives per USB port ?

**Reply by lrlbh** [reply]
![0f5e203c2e1cfe02db4175c0ea0e1aa](https://user-images.githubusercontent.com/47717790/123240641-0df22500-d513-11eb-99e6-b37a0ea874ea.jpg)
![7e1f33e12a532dba98c6648424d8375](https://user-images.githubusercontent.com/47717790/123240579-003c9f80-d513-11eb-9ce5-82f2b817b20d.jpg)
![74a38f1536e6930f3797efd34b530dd](https://user-images.githubusercontent.com/47717790/123240592-02066300-d513-11eb-8754-4435757b23be.jpg)

use USB3.0
One USB port has seven hard drives.  
One USB port has 16 hard drives.  
Next, I'll connect both USB ports to only 7 hard drives to test if the problem is coming from too many hard drives.  

使用 USB3.0 
一个USB端口有7个硬盘。
一个USB端口有16个硬盘。
下一步，我会把两个USB端口都只连接7个硬盘，来测试问题是否来自过多的硬盘。

**Reply by loppefaaret** [reply]
it is very possible that even if you got 2 USB ports on the machine, both those ports go to 1 chip inside, and is working together, so it acting as 16+7 = 23 drives on 1 chip

**Reply by lrlbh** [reply]
Just now I closed 8 hard disks and the situation did not improve. Now the hard disks mounted on both USB ports are the same, so the problem has not been improved.
I counted the total number of times the problem occurred over a period of time.
Some plots have been delayed for up to 7 times. Should I remove plots that have been delayed for more than 4 times?

是的我听说过这个事情多个USB端口使用一个芯片的问题。
刚才我关闭了8个硬盘这个情况并没改善，现在两个USB端口上挂载的硬盘是相同的，这个问题并没有改善。
我统计了一段时间内，问题出现的总次数。
某些 plots 出现延迟的次数达到7次，我是否该删除延迟大于4次的？

delay sum 432
A1\: 37
A2\: 36
A3\: 48
A4\: 4
A5\: 2
A6\: 33
A7\: 30
A8\: 32
A9\: 55
A10: 3
A11: 34
A12: 40
A13: 5
A14: 39
A15: 34
-------------------------
A1\--------96d5aaf847d1875d38f3f9ba9a6e300242802629 --> 2
A3\--------55d2182be50a6370be6dfa1271cf32cea6309785 --> 1
A11---------8739be998b8b0ad33932f9ff447a4f08bd3144d --> 2
A11---------cd3b762e78b7b5ac9e4ea191c4bb029f5370f5f --> 2
A8\--------3bc91ed80e6540073b70841e0c40816ba13d2106 --> 1
A12---------09cafa2eb33c87801982e108657774fde997fff --> 2
A13---------ef9ce9319565e23533474ae7582303ccf1c3cce --> 1
A2\--------d5977a8271eeb28b41b7d37e3e8a2eca8d1e4779 --> 1
A1\--------de8def5d35de87f00dd315aad36215e9474df1f7 --> 4
A1\--------1f0b8de32810b0df67836850d58eb570eac6086b --> 1
A14---------1f6e8c06119036124d9e6ada44b0b7918bff992 --> 2
A12---------9b49ece47d609ae69a0a35f4f8bdbeb10a94978 --> 2
A11---------353e1615886282ef1bd4127b949c73235b8fbe4 --> 1
A3\--------76301ebcf0d71e450cd38563a1a9ec1e858c6b41 --> 3
A4\--------465f6f8c1d88cf48aad0401bd17da9265a71c66a --> 1
A3\--------d7789bdbc5792a3ccfe9ce2902816e2cf682fc03 --> 2
A12---------612d1be94e91864657a8ca4e23bdb4d29548632 --> 2
A15---------a7ce448c4ffba01452b95bd42ae7f6f3ef9f4fe --> 2
A8\--------2401ab30dd15b436c824d5b7bced94f2b663345e --> 3
A1\--------a228c5fc13f70d8927712cca9b4de8d4a2f45edd --> 2
A8\--------41e26be5ea5d22c0b0c764d30e90a96b788dcc21 --> 1
A3\--------fc0f7959a4dd18447b731f1d83569035258e9e98 --> 2
A14---------8f42b748d5de4d8c525a7a61474226e5f2d92f7 --> 3
A11---------1c20a6bf1dc4fd02de0e5b8c8c10bfe5c1bf403 --> 2
A1\--------61cd3058fd2eb6cdbd596bd32181f835885f5ee9 --> 1
A2\--------034db1ad38e59e20d36752b695d592e92cb25435 --> 1
A1\--------3600d0689b3c0c185de759d102a5fe7d7358a920 --> 1
A12---------f5e7225464fdca6e46c3aee23737b3a42615121 --> 1
A9\--------23204a3565f0104e005512e347b326e662a95bad --> 2
A12---------b54c5cb21c76f634eeca2c0459c1107feea18d1 --> 1
A13---------e35de39b9438818f286951a883b70a3df444b06 --> 1
A7\--------19be5bea25f83d7d523df742eee5c8a8b7d3f2c1 --> 3
A15---------d287415852683235b3bcc970bc9c927def70120 --> 1
A6\--------7d09460e49070da28968c820915f340087c7f8d9 --> 2
A3\--------3f43604c7ec772f333cb0c9615ba90e7f9b4753c --> 3
A3\--------6540aae16c7f67a364a25075268b9f8e68716ecb --> 2
A12---------c6bca93659d766d6764a41e39ef2fcbd1f88051 --> 1
A6\--------dacb0e5de3c212cb79ba965aea2298f2340dd5de --> 1
A15---------f3c68f78b126791ddd99e19fba0974505927dc4 --> 2
A3\--------495e63359db25b6c19982aff29f19380e3213686 --> 1
A6\--------9ba48ffd924787bda56641007ff2008030dd5d76 --> 1
A9\--------c2ce8aa987d9f8f25af94d2b9360e3df65e64eca --> 1
A12---------f20a1b8b199fed158bb3926f35931349eb52596 --> 1
A1\--------0bb3c6d4be9f6d2ab635c3714725173b890205bc --> 1
A11---------dbba815fdf8b744bad8835639604ffb011cb6c7 --> 1
A1\--------afcac7c267975948e5b7212a6b42d170b61ee792 --> 2
A1\--------12bff1123570639a26629d998a1fbf167c922234 --> 1
A3\--------eb5dca9d41e9ea612365c8562d613e7791424dc4 --> 3
A3\--------c170812df93323ec1a40cb2e43fbc684b50556fd --> 1
A6\--------d2fc625623dca60713d5b33145ed78efdb676889 --> 2
A8\--------a234f3cb0d47bf3b544ede93b2a6f41b9adbe193 --> 1
A1\--------96ce0648979d25b9cf27c8babce470cfd9c4f261 --> 1
A3\--------16bb3f519310bc1b04b860d8ab6e164482c7f699 --> 2
A8\--------aa6bdc37b5997128cd7c3cf243be8988307ebfe8 --> 1
A11---------33dfd815e984b66ac419abfa7a60e6a25d2040f --> 3
A8\--------3febdecdbd1b250a376dc43e83a1407f2aae309e --> 1
A8\--------876cd844ca5776d346f38410bbb1b58d225c1713 --> 1
A7\--------bafe56c3f5a519499b353000cd74ac7e8c22c9f9 --> 1
A7\--------4abec2d02b8b138c1702c8c8099de8e301e39ccf --> 1
A11---------9f5909d07380babab32b2a2ad9fcb9f2ae04773 --> 3
A1\--------22e033dbe1ea3c74d484639a7d5f3e9e647f65ef --> 2
A4\--------3455df422a5b825a6d3cb91639198c28dc329fff --> 1
A8\--------e125647586b4a1c62ab87bfd89f03dfe0c70a903 --> 1
A6\--------b6eaac7ac09a2a66103916cd3c25a22d51b9d492 --> 1
A1\--------13bc5f2b985df951839bc9eb270382de4e72ee7a --> 1
A9\--------90bde672fb0b0fbffd0f6569eaf0b7db9d7d28e9 --> 1
A14---------98e044fddc2f8f63bacf1efb09cefe5f74429e9 --> 1
A2\--------5d7d3fba1de49d688ab2636d092559fb0fb77b7d --> 4
A10---------201b00d22caac3f4fca51a8910fa0f5ce15152c --> 2
A8\--------07453efa97acd25ff2d9b4349f2bd3cdefbd5683 --> 1
A15---------0b12d1055e729ef1a1ddfbd6ee797ec1cea71ed --> 1
A8\--------082d9dbbf917635f289f5907fe6c724c5d33a3d1 --> 1
A6\--------0e2b9a4bb7f0f197bfaf1a05e66a59279e57a9fd --> 2
A15---------9c43789c82a9cd938a9926ff3086de1d1b88444 --> 3
A14---------e36627035b9445e2ac6ecb5210d434b88dcc55b --> 1
A6\--------7f35e6d33d5909e532f21169d66b8c76d4a11f05 --> 1
A1\--------415a67749cb600474a2a684b67f266603fc3bc65 --> 2
A11---------ca30f1b8e6e2f91c72435d002849d2ee39c3007 --> 1
A14---------76eb132490428decb360c338726dc928382242d --> 1
A7\--------a2f9ea37058b3edcc82370e65f94d6d389102415 --> 1
A8\--------ffa6bbf9b1e13ea3ee03ba17eb238e99f03a6132 --> 1
A11---------3f861d5b876285008b1d7588b43e064ff58460c --> 1
A7\--------d8d0a27df19352c9b01b691912231971d3931300 --> 1
A6\--------1a494669857b8bd5609f15db28c621cc5d994466 --> 2
A12---------895dc0f99dee9ecc2c46fdb456ecde854f26b01 --> 1
A15---------68d31e080727c0712b55492762d99a9fc3a54e0 --> 1
A14---------3486b28398fce64dca2a3607b936bddfab2aca7 --> 1
A7\--------ac441b30112d9dc41881af0ba15d0ae3763b2a12 --> 1
A6\--------6bfcfcbb1d94083891754d999b53e5f6d6892201 --> 2
A3\--------04990a8f4869bc788b2da7f53ae572bdc89c1bed --> 2
A6\--------97d3156d840b6267ae10640c1f1d6563150fc25e --> 2
A9\--------c78019dcf4392afc2ffd486d9bedeced6d9075ff --> 4
A11---------8b99854a4c0c1913807f39768f80f114038e321 --> 1
A9\--------a6b28776997f679305b1104810afd340a8038013 --> 7
A12---------bb12c755be9c5d7a94ce104d79196ea72c9c894 --> 2
A13---------897fee851bdd9c91c676f5507e2e23630c2a462 --> 1
A2\--------23ef164e6eba569f6857c90282dffde2f3949310 --> 2
A8\--------860a7c0d8b543883b8e239ad95a1488f92b5e3a2 --> 1
A3\--------17e20c0150b484a3ffe0a9cea779a6a36e2cf210 --> 2
A5\--------f7e1799e61a4e8fb7bd1f2849b5c067d4810aa73 --> 1
A4\--------5addf6c909ec98c0c2754f78070d07df2946de80 --> 1
A8\--------6515e924ca244737de95fc69610cdcddf051a02a --> 1
A6\--------3c7b4c412d66e54fe1dddb5e1866bedf0819c688 --> 1
A6\--------1781cccd57201f7360579e5ad5be1d5f155419db --> 2
A1\--------daf1f9f0f6460f41048e11c990009e27bb842106 --> 1
A11---------4e188fde810a899ec59d5d3597b01bdab87b5a3 --> 1
A2\--------cb437993ff1f8ccb3b17575134928d1057bec383 --> 3
A6\--------a0b6fb403888435671124172eb9ef8d0ce9a39ae --> 2
A3\--------87e2a254b9919868f29ede65ca560ccffab01bf6 --> 1
A15---------647f2b827b40bb5c0fb056a68da9d0c6ebbbfd5 --> 2
A8\--------ba7e4a095d976f8da8a503c3112dbb5fe64fb8a7 --> 1
A2\--------2bf6782f0431dc8960c854fb08fa1cb00a31ac2d --> 1
A9\--------b64d49a893ebd57492ae611b943042cccc46fd75 --> 2
A6\--------99755d97943e288fb60411f069415d485d7eef92 --> 1
A7\--------cc9822ef2a3781819861d5251b6570ff4a0a5360 --> 1
A9\--------9b04ff82c17a79515ea7c081ef278d2effb88580 --> 3
A7\--------62b492a22e1b494790d8211369b4b830e1fb7396 --> 1
A7\--------064ac89e49413ad8d4af05695fbe268e90fae0ad --> 2
A11---------1350080bae04ae0dae6f22fc741c9c3c0ffc39e --> 1
A8\--------562a818bcd0a54fa40d582a976bd1e0781b04bb1 --> 1
A8\--------f736d6251c4fc27a781e618bdb4d1c29b6cd34c4 --> 2
A11---------32e894f9909155b378171677ce7bf022ea9ccb6 --> 1
A1\--------06ef325fd65916789a50cf00edf795b39cfb3eb5 --> 1
A9\--------e53c48f641d0b5df52a06aea449354e60b627535 --> 1
A15---------10941e13db4b66cf0804b35bed2a231051e3124 --> 2
A7\--------78540bc3d05a49f29b5f9e76b2de3d2c985a65df --> 3
A9\--------d042f0c1819d797169a009a639ac6a88f49455dd --> 2
A8\--------557c9e2f6fd5f59a83b59bb18b184b422e641aa7 --> 2
A10---------2914cbc6aa289238d57fa73522bc9337195e875 --> 1
A15---------11ec84d7d13ec7cc15715ab5db82fe883cba892 --> 1
A1\--------6e24bae5943a4ebdbceac9dff930596c745d14e6 --> 2
A15---------5adbd48eec3f5dee9836664f26962bae4b1e0dd --> 3
A14---------01bd1914a547a55e731db37cd3125a04938d7de --> 1
A14---------055280b86a44bcef22f855678f3adbdbef267a5 --> 1
A7\--------a0856c79023e6f91fdd2aa6ae950ef0b55deb613 --> 1
A3\--------38424ab6c5b20f2d343dcbde58124dff4c934dca --> 1
A2\--------5873ae65e8194cd36fbcc638647f97783ff46519 --> 2
A15---------34b12d97b7a3640c5f1e3a1604d75d67d37238f --> 1
A9\--------1cb96b32d17aa5291721e72d2331312709d2627a --> 4
A12---------05b43e315fe80c2f5d0d61307d571f302046587 --> 1
A3\--------456bd1367825614fde63525c5cf10b5d8fb1952c --> 1
A9\--------7468881f4d5877c6e9c60a31c06ab8e7663f9f73 --> 1
A9\--------4f9f17dc7a2bcff5abba488b06739045522eb606 --> 1
A7\--------1a51004c15033f35e026fc7baf86f98156a6f470 --> 2
A12---------c26346fe9a8fb40367aa0e22e2897126f1214ca --> 1
A12---------c776c81e9411104fcdcd1cc2935ca650f2ab521 --> 1
A9\--------7cd04a19dc9c6de46c675dcb6edfb4c96e234314 --> 3
A11---------768d3a038c9397257760b01a78ac762c4109be6 --> 2
A8\--------da03c1201cf76680eb6b75a860678297d66bf534 --> 1
A15---------348d17c37a940a49559d12a3afd48437f8969de --> 2
A12---------994573074e479603bc1b8ad808676eebe8363d0 --> 1
A14---------fa31ffe8e8e6116d3ca4e906ae4b0b5e897944f --> 3
A3\--------5189f98958d989a27d132c3dc6ce5cd7404d6163 --> 2
A3\--------e8aeb862bfb2532bfb8a30a0fe9e01577abd24a8 --> 1
A11---------7b7a869593b8626dce395dd07749bc3f1e1bd83 --> 2
A15---------6ca53c86511ae827aae4f11d6d007bc06b53180 --> 1
A6\--------922cab9fbaced3bf41894e9530940f6af7a52896 --> 1
A5\--------1d2b36399ab97178a479da7ef99c1ebe201c16e9 --> 1
A15---------5bc9f27fd5648349a5f7d448c591132debedf65 --> 1
A14---------43aa57550b496e22e0353f9e9f730fb52c4b8d9 --> 2
A7\--------ad577665e6be6b8480b1894e4142cb06d1127e90 --> 1
A9\--------8ce2b4e75fe4a11bd3c06831b5679791433e922e --> 3
A2\--------661813f1fa03d4296df04e0a6714b9be4e7ef571 --> 2
A3\--------a269e1556e08a425779787da6fb80afe0a428805 --> 1
A14---------f114646e885d5ac799961581181b3cad422edbb --> 1
A3\--------e3a9416141cea27c0219a4de2ec08ccb9e65e204 --> 1
A1\--------cdf59ed39472867a2fea5b7f57950a90180d9de2 --> 2
A7\--------b49bc2dc26d73cd55179179f00fdf3f2526ff305 --> 1
A14---------cf0095833ea8e3e39222daf56c9beb346ca4ac4 --> 1
A14---------998c7b8097f59e42657a0a56f74f9cc1c4f92cd --> 1
A14---------6af2012787f4ea2e5205e8754c8a82e796c1d6d --> 2
A11---------15e58b560f1524374fcaed4e8704412244ccdf7 --> 1
A12---------2582b0362d670ef4762b8c946f7027483609fe4 --> 2
A7\--------def598f92c1494f09d7ecd594f766b6baf264e85 --> 2
A9\--------3c14f3c74f2c58db8d8f0acd122f3bb11e9105b6 --> 3
A9\--------a1a01eaa108346b2b2666f2a94216401d5e50b8b --> 1
A9\--------fc42149a3e8a7a8c8f02dabbefd2181c3eb49ed7 --> 1
A6\--------cef66df9b66c7f6587664190e66745772791074e --> 2
A11---------366a008180d4ce5555785cda1660578dcfef704 --> 1
A3\--------96fb635b533c540630d8e1e28631a2ef88191e0d --> 2
A2\--------9834083b41de9ca895b6456984013ecf6e3b9090 --> 1
A1\--------d16253427919bba679f300577eebfb5487e6d7e7 --> 2
A9\--------722a63cb5de3d1728c0f10ac32891ac484c818ee --> 3
A12---------48cb9f34c6bfbce9fcb6b582f93d15e2f4a2eda --> 1
A9\--------bb8ebe8986c8ce5c417543108c945453e96799ec --> 2
A14---------814397b7973b8e15c8c118f21545f9631d736d2 --> 2
A3\--------a0aa4df5d738d362665b6c8595d5059aae604b3a --> 3
A12---------89316f6422717dbf2099e88e74d0502b85f9c22 --> 1
A2\--------be6db52986a30cc01d5f71f5ddd5ddeb378fee47 --> 1
A2\--------d255d58b3e75f4e952858e164b3ef21f26ffeee3 --> 1
A14---------52605ccd725b666adc698ab2d0c2d8060f48804 --> 1
A14---------b1cf815be805c0632edaaaa6a961e87452cbf91 --> 1
A14---------70d4b495838af5ad7b0ca1be3c31c6e9579ace8 --> 3
A15---------3db8a733c99732aea97db098dea0008ef8e4ea0 --> 2
A3\--------88283de225cec32c0823aed3a2da0080a82a7568 --> 1
A11---------9467487fc52147df52a779176e5b4e5f840c21f --> 1
A7\--------e95233e7c6919e0556f767c2e7326cbfb5a44875 --> 2
A15---------41dc92586b30cd0056457dc9f5f2124a8ac4cac --> 1
A8\--------40ceabeb536a3f2278406678e0ba51fcdb649a7c --> 1
A15---------069b6e5158947b095d0f658adde22d3d61bd666 --> 2
A12---------9bf24fb81d260e3e69faa4bd498e97604cce6b4 --> 1
A6\--------a320164ebf100d263f9fe4a3065ad3dcfc58c3c7 --> 2
A13---------fc675ce0e6bb1a961b669f58e571ddbe5457581 --> 1
A2\--------c73416215665bc74e8ad39bd1972b7d3410ae017 --> 2
A9\--------86a2c1d02c2db66f7d80d2030be10f94cee635d6 --> 2
A13---------16827984144b7f8a1ec27afe8a4d40aac1e0399 --> 1
A8\--------8681854b82bc164e6bcc6d6b21c252a5c4390e90 --> 2
A9\--------39831a8aa8d3fa5b8d21c7aa59a61faedbec1d1e --> 1
A6\--------1f3fa9fc5181ac5f03e66162db2f5bf44ea1e5c7 --> 1
A9\--------057b02dbe596d6f413f7293c13134f94d0ebac43 --> 1
A15---------ddf6c3593ea9c632100fc844155b2c8fc418263 --> 2
A6\--------7cca550c559b29ccebef1968281108e35da999fd --> 1
A1\--------4dd2acc0aef42ba31773f5477b4f507461e7a351 --> 1
A2\--------e4bd504afd37ceb5fd4a34e1748d04e4f4f5028a --> 1
A14---------29635d1b1bb40885c55d9fa6a5c033bfd274ff2 --> 3
A3\--------81a8145a39ab80b08f044c939ed9d152f2206c82 --> 4
A11---------88f2f1daeae50d606a863c5efc56a5aa66dafab --> 2
A12---------bbb8363de039c0404fb233280a5ef51f7243618 --> 1
A8\--------01b77805819c42f5f997f5634c8e1a598f3465ee --> 1
A3\--------31c6603ffe9adacfcc3e0583a3b7353a8ca392fd --> 1
A15---------8c4e01fc81c7c0b91cc781e32eacf973e58f789 --> 1
A1\--------9bcf57aa364f847f6b2a471eecbe1ffe8bff193d --> 2
A3\--------70d5883b7cfcb961beb23b6e16c7573b9e16caa6 --> 1
A2\--------0cc30f58d4141d9c2b7d81646b8920bb21bb3cfa --> 2
A11---------425ba1b4aae9eed249caa13a5b9edb307a79f3f --> 2
A12---------f40590c9013701b95b3687c4b06f20530e77770 --> 4
A7\--------790e76d44f1a11bed19b31a8ce6dfbdd271e8830 --> 1
A14---------609ecbefb3cb9fb1f0a50dd89d440107120743b --> 1
A2\--------b0208bbe44ea5a150655fdc2c5b51028071cfeb1 --> 3
A1\--------e288136f1e6a535603515ceccf39c0fc637d1e22 --> 1
A2\--------feea65b4114f0b93a3c33e5814cb9676888aca90 --> 1
A3\--------7d7641df22bf5457884b2e8b687e8b8fcc1adf8b --> 1
A2\--------f44ce7316f8ee41f198f9897118d4d60b18975c8 --> 2
A8\--------1f06d866690290f0df6d23e1a75605333454ed86 --> 2
A8\--------6303e9f2da150105b0c4692571abbb0eb2fc6209 --> 1
A2\--------4ae41345c157b4ce16f680f5de0fc9bc8cef51bf --> 1
A1\--------72b04f9914da0e8282301ee3baf63d9720c868dc --> 1
A12---------8f7fc928faeb7ebc7e8e892e25769794e6689d0 --> 2
A3\--------8e91eebf9cb8e447a150e4bb81c32d99d90449ae --> 3
A12---------599ef88034452aafe251a5afc1069dda498a128 --> 3
A2\--------0d99b8c2ac1f3d2ee8f249afd231bbae22662881 --> 4
A11---------32c9814bce1b8ee30517bf2f15c083f87ffdd46 --> 2
A9\--------e60b97e101706251702386874dfa33a05f18aa49 --> 2
A9\--------fd7fa4e3c2e8b97886a9bf70ba727982d1dc4a5f --> 4
A14---------fe3eb79e871726d207194c339be414e03f9ad02 --> 2
A15---------8460adea8ba173f1fa49f523e22c13a6db8cf77 --> 2
A7\--------c82d90161f07e5550ffb4658ff3df85c841da14c --> 2
A12---------36ef15bc754430e95fd763f67fdb142389d909b --> 1
A12---------c50de44db4630e87d7afa1e0f79eba88aa2f2f9 --> 1
A14---------6117066f223e3b519c60b7555cb3fba259c5335 --> 1
A6\--------c05fa58c2fa7aed384bc83341f47fadec66d8fab --> 1
A7\--------9d55b08a0a2b226a42ca3cc15b359834a60300f5 --> 2
A6\--------c2774299af4dd45c5f582d2b782a745fe9c9e92d --> 2
A14---------bf7bae5f736eb025f6f63f5924a8c5a476232e2 --> 3
A11---------df1c3c78bd2d54a547ec0811742b49d03d27bba --> 1
A12---------4b1e076309b9dca34545e51d8e39f031592351d --> 1
A8\--------fcee2e73cdf9e87928a9d4d8b794ac13bf830833 --> 2
A2\--------8804db1901a019be38af1deb9c5e628b0969d947 --> 1
A6\--------fdca2a9f190ccc24e145ab2f1f2226ffbad65bef --> 1
A8\--------7502f20cef0adcfd95db7f0d9816e1231fb554ce --> 1
A12---------6167f24db6ec6b9b837b60b2ecd554c41dd05c2 --> 2
A12---------7975a68e5fbee4bae8e1b310240803330747107 --> 1
A4\--------a47d835ef7cfb979d0997af62477597ec85a2c07 --> 1
A1\--------c077dc995652277a46d929b2d69b378700f5a60b --> 3
A7\--------d13387e96be0bfd6c78aea0e8876ba4ed1f99f8a --> 1
A15---------256be40c77451ad49eee7eb6f086d487dcfedec --> 1
A14---------6c8f2291ff9d59739190139605459e921d94a5a --> 1
A12---------d722c44d5d5e7797991966621b26149b83683f0 --> 2

**Reply by lrlbh** [reply]
I don't have enough STAT interfaces, otherwise I should be able to further determine what the problem is.  
I now focus the problem on two points:  
1, hard disk problem  
2, easy to drive line and hard disk box problems  

我没有足够的STAT接口，否则我应该能进一步的判断出问题出在哪里。
我现在把问题定位2个点上：
1、硬盘问题
2、易驱线和硬盘盒的问题

![16b62adeee219227068c1cb5d4def50](https://user-images.githubusercontent.com/47717790/123248763-e0a97500-d51a-11eb-9ad0-c4d6e5fcd737.jpg)

**Reply by laisangbum** [reply]
我也想是硬硬盘问题，因为有plot可以，有的比较长时间

**Reply by lrlbh** [reply]
如果问题来自硬盘那么，应该是健康度过低，温度过高，硬盘本身就不行。理论上来说我的A系列硬盘是要比移动硬盘好的。

**Comment by laisangbum** [comment]
我也使用usb连接，也有这种问题，我的也有出现20秒以上。查看debug没有写是哪个硬盘，哪个plot。我想也是硬盘的问题或者是usb autosuspend， 我是用Ubuntu server， 有的ntfs 跟ext4。有使用usb hub 32 孔， usb 3.0

**Reply by lrlbh** [reply]
应该和休眠没关系，前面我尝试过一直唤醒硬盘。debug文件在  ~/.chia/miannet/log/debug.log

**Reply by lrlbh** [reply]
查看的时候使用 | grep 筛选 took，可以看到具体文件和硬盘，你查看的应该是INFO信息

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
